lib/task-data/tasks/install-coreos.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     implementsTask: 'Task.Base.Os.Install',
     options: {
         // NOTE: user/pass aren't used by the coreos installer at the moment,
-        // but they are required values
+        // but they are required values for Task.Base.Os.Install
```

Comment:
Yeah, this is right. 

This was actually the original reasoning behind why we had base task definitions be separate from jobs, so that you could have different schemas/required options for the same job, but in practice, it seemed to be mostly redundant rather than useful. But I wonder whether it's an idea worth holding onto when we refactor base task definitions into the job classes.